src/partition/growfs.c

```diff
@@ -241,6 +241,10 @@ static int run(int argc, char *argv[]) {
                 return log_error_errno(errno, "Failed to query size of \"%s\": %m", devpath);
 
         log_debug("Resizing \"%s\" to %"PRIu64" bytes...", arg_target, size);
+
+        if (arg_dry_run)
+                return 0;
```

Comment:
I'm curious: do we do dry runs silently or log about things being skipped because of it?

Reply:
On https://github.com/systemd/systemd/blob/2af239c20e024a77b800d9785643f2a0fe8c8cc9/src/partition/growfs.c#L71 it also silently continues, but it does make more sense to return a "stopping due to dry-run" message.